app/Livewire/Project/Application/General.php

```
@@ -622,6 +622,7 @@ public function updatedIsStatic($value)
 
     public function updatedBuildPack()
     {
+        $originalBuildPack = $this->application->getOriginal('build_pack');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$originalBuildPack'. (undefined)

(UnusedLocalVariable)